tests/test_config.py

```diff
@@ -231,16 +224,6 @@ def test_reload_config(self):
         config.register_extra_parameters()
         config.load_from_file(LOCAL_CONFIG)
 
-    def test_custom_experiment_module_set_and_retained(self, reset_config):
```

Comment:
Not sure why this was deleted...

Reply:
This test was a bit weird. It assumes that Dallinger loads the experiment module when calling `config.register_extra_parameters`, and then runs some tests on what happens when you clear the config... But it relies on the fact that original Dallinger tries to import the experiment module even if there's no experiment to import, which doesn't happen any more. I tried writing a revised version of the test for the new version, but it ended up breaking other tests too... I think this test is unessential enough that we ought to just move forward and drop it.